pkg/components/scheduler.go

```diff
@@ -283,10 +283,18 @@ func (s *scheduler) createInitUserScript() string {
 	return strings.Join(script, "\n")
 }
 
-func (s *scheduler) prepareInitOperationArchive() {
+func (s *scheduler) prepareInitOperationsArchive() {
+	initOpArchivePath = "/usr/bin/init_operations_archive"
+	// COMPAT(omgronny): drop once new srcipt name is everywhere
+	if _, err := os.Stat(initOpArchivePath); err != nil {
```

Comment:
Also you can reduce the number of ifs: `if _, err := os.Stat("/usr/bin/init_operations_archive"); errors.Is(err, os.ErrNotExist)`